src/zenml/zen_server/routers/workspaces_endpoints.py

```diff
@@ -349,6 +351,68 @@ def create_stack(
     )
 
 
+@router.post(
+    WORKSPACES + "/{workspace_name_or_id}" + FULL_STACK,
+    response_model=StackResponse,
+    responses={401: error_response, 409: error_response, 422: error_response},
+)
+@handle_exceptions
+def create_full_stack(
```

Comment:
Why is this endpoint needed actually? Is this just to implement this quicker in the frontend, or some other reason? It seems like one could achieve the same thing with the existing endpoints

Reply:
Yes, quicker for the frontend, but also easier to register the whole shebang with one API call in the AWS cloud formation stack, where we only have access to very basic python libraries (urllib) and the python script needs to be supplied inline. Both reasons why having an endpoint behind which we can hide most of the logic that would otherwise be duplicated in the CLI, front-end and cloud stack deployment templates.